webapp/app/.server/service/l2-cancellations.ts

```diff
@@ -33,6 +44,26 @@ const eventSchema = z.object({
   }),
 });
 
+function blocksInADay() {
+  if (env.ETH_NETWORK === "mainnet" || env.ETH_NETWORK === "local") {
+    return 24 * 3600; // 24 hours, 1 block per second
+  }
+  return 24 * (3600 / 20); // 20 seconds per block.
```

Comment:
Can we do it with a switch/case? This way if `env.ETH_NETWORK` is modified, adding a new network in the future, this won't just assume it's 20 seconds per block, and will throw a type error (as `blocksInADay()` will now return number | undefined)

```suggestion
    switch (env.ETH_NETWORK) {
        case "mainnet":
        case "local":
            return 24 * 3600; // 24 hours, 1 block per second
        case "sepolia":
            return 24 * (3600 / 20); // 20 seconds per block.
    }
```

Reply:
This is going to be applied on next release